src/stdhasher.cr

```diff
@@ -0,0 +1,248 @@
+require "secure_random"
```

Comment:
SecureRandom is stdlib, not corelib, it shouldn't be injected into prelude (indireclty).